aws/templates/id/id_spa.ftl

```diff
@@ -195,7 +249,8 @@
                     "Id" : formatDependentWAFAclId(cfId),
                     "Name" : formatComponentWAFAclName(core.Tier, core.Component, occurrence),
                     "Type" : AWS_WAF_ACL_RESOURCE_TYPE
-                }
+                },
+                "paths" : pathResources
```

Comment:
needs to be conditional so component deployment correctly detected

Reply:
fixed